src/sagas/wallet.js

```diff
@@ -325,7 +342,7 @@ export function* listenForFeatureFlags(featureFlags) {
       const oldUseWalletService = yield select((state) => state.useWalletService);
 
       if (oldUseWalletService !== newUseWalletService) {
-        NativeModules.HTRReloadBundleModule.restart();
```

Comment:
This removes the need for `HTRReloadBundleModule`? Because we have an implementation of this module in both java and Obj-C, maybe we can remove those files.

Reply:
We can! Thanks

9172518d9e80c9ba05703fdf4b18285ca20ab958